plugins/anchor-links/index.js

```diff
@@ -50,6 +57,12 @@ module.exports = function anchorLinksPlugin({
 
 function processHeading(node, compatibilitySlug, links) {
   const text = stringifyChildNodes(node)
+  const level = node.depth
+  const title = text
+    .substring(level + 1)
+    .replace(/\(\(#.*?\)\)/g, '') // Strip anchor link aliases
```

Comment:
I might lift this value so it can be shared: https://github.com/hashicorp/remark-plugins/pull/38/files#diff-fe4c013e484c4d3ceab608a44afc8f5118475bba717203c7edffd253c2a4d2abR203

Reply:
Could you re-link this? I think when I pushed more changes just now it broke your link.

Reply:
https://github.com/hashicorp/remark-plugins/pull/38/files#diff-fe4c013e484c4d3ceab608a44afc8f5118475bba717203c7edffd253c2a4d2abR200